src/util/Spotify.js

```diff
@@ -124,7 +116,7 @@ export const Spotify = {
       let userID;
       
       /*This fetch is set to GET the user's ID */
-      return fetch('https://api.spotify.com/v1/me', {
+      return fetch(`${SPOTIFY_SEARCH_ENDPOINT}/me`, {

```

Comment:
why not rewrite `savePlaylist` to async await pattern?

Reply:
I am going to. Just didn't have the time yet. :)

Reply:
So I just finished my attempet to refractor the `savePlaylist()` to the async/await format. It got stuck on the `try` on line 153. I traced it back and found that while logging JSONResponse to the console it appears as "native code" and nothing more.. therefore `userID` stores `undefined`.
The userID is used in the next step's `POST` request (line 164) as part of the URL the `fetch` ties to contact.

Do you have any advice on what should I do with it?